<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BornDialog</class>
 <widget class="QDialog" name="BornDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Congratulations!</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icon.png</normalon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#BornDialog{background-color: rgb(255, 255, 255);}</string>
  </property>
  <widget class="QLabel" name="SpiritImgLarge">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>240</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="BornHint">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>340</width>
     <height>40</height>
    </rect>
   </property>
   <property name="pixmap">
    <pixmap resource="../assets/UI/resources.qrc">:/HatchingHouse/Born/new.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="YesButton">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>330</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="icon">
    <iconset resource="../assets/UI/resources.qrc">
     <normaloff>:/SpiritUniverse/yes.png</normaloff>:/SpiritUniverse/yes.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../assets/UI/resources.qrc"/>
 </resources>
 <connections/>
</ui>
